docs/visual-basic/misc/bc41008.md

```diff
@@ -25,7 +25,7 @@ The preferred way to specify a file that contains a public key for an assembly,
 1.  Use the [/keyfile](../../visual-basic/reference/command-line-compiler/keyfile.md), [/keycontainer](../../visual-basic/reference/command-line-compiler/keycontainer.md), or [/delaysign](../../visual-basic/reference/command-line-compiler/delaysign.md)[!INCLUDE[vbprvb](~/includes/vbprvb-md.md)] compiler options instead of the <xref:System.Reflection.AssemblyKeyFileAttribute>, <xref:System.Reflection.AssemblyKeyNameAttribute>, or <xref:System.Reflection.AssemblyDelaySignAttribute> attributes in your code.  
   
 ## See Also  
- [How to: Create Signed Friend Assemblies](http://msdn.microsoft.com/library/f5542300-58b4-4e1c-b809-8df11e95e69b)  
+ [How to: Create Signed Friend Assemblies](../../programming-guide/concepts/assemblies-gac/how-to-create-signed-friend-assemblies.md)  
```

Comment:
The link is broken. It should be (../programming-guide/concepts/assemblies-gac/how-to-create-signed-friend-assembies.md).